.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 15
+      - name: Cache Node.js modules
+        uses: actions/cache@v2
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Copy Cadence files
+        run: npm run copy-cadence-source
+      - name: Publish to NPM
+        run: npm run publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
```

Comment:
I don't know if e.g the JS SDK has this set up, so maybe sync with James and Jeff how this could be done. For the NPM packages in Cadence I just built and publish them manually, but regular / automated publication seems like a great idea, I might copy this 👍